osbs/cli/main.py

```diff
@@ -339,21 +339,57 @@ def _print_build_logs(args, osbs, build):
             return -1
 
 
-def cmd_build(args, osbs):
+def _print_pipeline_run_logs(args, pipeline_run):
+    user_warnings = UserWarningsStore()
+    pipeline_run_name = pipeline_run.pipeline_run_name
+
+    pipeline_run_logs = pipeline_run.get_logs(follow=True, wait=True)
+    if not isinstance(pipeline_run_logs, collections.Iterable):
+        logger.error("'%s' is not iterable; can't display logs", pipeline_run_name)
+        return
+    print(f"Pipeline run created ({pipeline_run_name}), watching logs (feel free to interrupt)")
+    try:
+        for line in pipeline_run_logs:
+            if user_warnings.is_user_warning(line):
+                user_warnings.store(line)
+                continue
+
+            print('{!r}'.format(line))
+    except Exception as ex:
+        logger.error("Error during fetching logs for pipeline run %s: %s",
+                     pipeline_run_name, repr(ex))
+
+    if user_warnings:
+        print("USER WARNINGS")
+        print(user_warnings)
+
+    return _display_pipeline_run_summary(pipeline_run)
```

Comment:
I would prefer calling this methods separately and not having this one as return value.